### PR TITLE
Added S3 example into example.yml

### PR DESCRIPTION
**Pull request by jwenz723**
Signed-off-by: jeff.wenzbauer <jwenz723@gmail.com>

### CODE REVIEW

File: example.yml [hunk 1]
```diff
@@ -141,6 +141,20 @@ metrics:
   aws_dimensions: [NodeID, ClusterIdentifier]
   aws_statistics: [Average]
 
+- aws_namespace: AWS/S3
+  aws_metric_name: BucketSizeBytes
+  aws_dimensions: [BucketName, StorageType]
+  aws_statistics: [Maximum]
+  range_seconds: 172800
```
Comment (brian-brazil) [inline example.yml]: This deserves a comment

File: example.yml [hunk 2]
```diff
@@ -141,6 +141,20 @@ metrics:
   aws_dimensions: [NodeID, ClusterIdentifier]
   aws_statistics: [Average]
 
+- aws_namespace: AWS/S3
+  aws_metric_name: BucketSizeBytes
+  aws_dimensions: [BucketName, StorageType]
+  aws_statistics: [Maximum]
```
Comment (brian-brazil) [inline example.yml]: Only max?

Comment (jwenz723) [reply example.yml]: Changed to Average only, according to S3 Documentation: https://docs.aws.amazon.com/AmazonS3/latest/dev/cloudwatch-monitoring.html

File: example.yml (hunk 1 above)
Comment (jwenz723) [reply example.yml]: Comment added

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**